baseband/vlbi_base/base.py

```diff
@@ -355,7 +371,7 @@ def seek(self, offset, whence=0):
         elif whence == 2 or whence == 'end':
             self.offset = self.size + offset
         else:
-            raise ValueError("invalid 'whence'; should be 0 or 'start', 1 or"
+            raise ValueError("Invalid 'whence'; should be 0 or 'start', 1 or"
```

Comment:
The standard in python actually seems to be not to capitalize the first letter of an exception string (because there is a colon in front, not a period). E.g.,
```
hash([1,2])
TypeError: unhashable type: 'list'
```

Reply:
I see; this is apparently also the case for `warnings.warn`:
```
warnings.warn("Test warning")
/home/cczhu/pythonenv/ml3/bin/ipython3:1: UserWarning: Test warning
```
Though both capital and lower case are used throughout baseband.  I've made the change in the files I'm currently working on, but I'll also raise a tiny issue that this should be standardized for 1.0 (just so I don't have to search through every file for errors for this PR).

Reply:
Yes, I only became aware of this relatively recently myself, and so the package is quite inconsistent. Obviously, not that big a deal!